CHANGELOG.md

```diff
@@ -1,3 +1,71 @@
+# 84.3.0
+
+This minor release adds new theming extensions for Lists and minor bug fixes and documentation updates to multiple components.
+
+## New features
+
+Theming extensions are now available for Lists:
+
+```swift
+let cell = MDCSelfSizingStereoCell()
+let scheme = MDCContainerScheme()
+
+cell.applyTheme(withScheme: scheme)
+```
+
+## API changes
+
+### Chips
+
+#### MDCChipView
+
+*modified* class: `MDCChipView`
+
+*new* property: `mdc_legacyFontScaling` in `MDCChipView`
+
+### List+Theming
+
+**New extension.**
+
+## Component changes
```

Comment:
Please remove "Component changes" since the section is empty.

Reply:
Done